BaseApi.Tests/V1/E2ETests/ExampleDynamoDbTest.cs

```diff
@@ -0,0 +1,77 @@
+using AutoFixture;
+using BaseApi;
+using BaseApi.Tests;
+using BaseApi.V1.Domain;
+using BaseApi.V1.Factories;
+using BaseApi.V1.Infrastructure;
+using FluentAssertions;
+using Newtonsoft.Json;
+using NUnit.Framework;
+using System;
+using System.Net;
+using System.Threading.Tasks;
+
+//namespace BaseApi.Tests.V1.E2ETests
```

Comment:
Is this file needed since it's all commented out.

Reply:
It's is there as an example of how to write integration tests if using DynamoDb. If Postgres is being used then the dev can delete it. It's all commented out because if not then the tests will not work out of the box without changing it the whole thing from using Postgres (which is still the default) to DynamoDb.